tests/src/FunctionalJavascript/ContactStorageExportTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_contact_forms\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\contact\Entity\ContactForm;
+use Drupal\contact\Entity\Message;
+
+/**
+ * Test Contact Storage Export form, corporate contact forms.
+ */
+class ContactStorageExportTest extends WebDriverTestBase {
```

Comment:
Why do we need a new class for this test? The name of the method is copy pasted and it can be generic: `testCorporateContactFormExport` where we assert only the right things are exported. No?